src/Common/MainController.h

```diff
@@ -315,7 +315,7 @@ public slots:
 
     QScopedPointer<Geo::IpToLocationResolver> ipToLocationResolver;
 
-    QList<Recorder::JamRecorder> jamRecorders;
+    QList<Recorder::JamRecorder *> jamRecorders;
```

Comment:
Hi @pljones , I'm very busy in this week, I will test this branch and integrate in master soon. Thanks for your contribution.